packages/mongodb-memory-server-core/src/util/__tests__/MongoBinaryDownloadUrl.test.ts

```diff
@@ -350,69 +401,7 @@ describe('MongoBinaryDownloadUrl', () => {
       ).toBe('debian92');
     });
   });
-
-  describe('getMintVersionString', () => {
```

Comment:
why did these tests get removed? it still could test if those distros are giving the correct thing - even though no dedicated function exists

Reply:
i removed them, because `MongoBinaryDownloadUrl` should never again get initialized with sth like 
```
 os: 'linux',	
 dist: 'Linux Mint',	
 release: '16',	
 ```
since it will take the values from linux mints upstream lsb which contains ubuntu properties.

Reply:
and what is when an other method is forced?
https://github.com/nodkz/mongodb-memory-server/tree/beta#options-which-can-be-set-via-environment-variables
> ```sh
> MONGOMS_USE_LINUX_LSB_RELEASE
> MONGOMS_USE_LINUX_OS_RELEASE
> MONGOMS_USE_LINUX_ANYFILE_RELEASE
> ```

Reply:
sooo. I think i made them a bit nicer, but after all I readded the custom handlers for elementary and mint.